### DISCUSSION TITLE
Option to render other .hbs (static pages) files like home.hbs

### DISCUSSION THREAD

**Discussion by claudemircasa** [GetPublii/Publii#1337] [Help] [answered]
Plublii is promising, but I'm creating a custom theme and it's not possible to add other static pages (contact, about us, etc). Is this scheduled for future releases?

**Comment by bobmitro** [comment, marked as answer]
You can achieve this by marking the post as hidden and/or creating additional post templates.

**Comment by claudemircasa** [comment]
> You can achieve this by marking the post as hidden and/or creating additional post templates.

I believe it is not the same thing. I would like to be able to create a "contact.hbs" file with custom code, for example, and have this template compiled as a static page in the root directory as "contact.html"

**Comment by bobmitro** [comment]
Instead, you can create a post template featuring your custom code and apply it to the hidden post. This approach will function as you've described.

**Comment by bobmitro** [comment]
In general, the post marked as hidden works as a "static" page.

**Comment by claudemircasa** [comment]
> In general, the post marked as hidden works as a "static" page.


How do I create this additional template for posts?

**Comment by bobmitro** [comment]
I recommend visit the dev docs https://getpublii.com/dev/how-to-create-custom-templates/

**Comment by claudemircasa** [comment]
thank's @bobmitro

**Comment by claudemircasa** [comment]
@bobmitro Can you help me with this?

![image](https://github.com/GetPublii/Publii/assets/7956750/dba18651-8643-47cb-90a5-a1082e1e5fab)
![image](https://github.com/GetPublii/Publii/assets/7956750/0f8b9052-de34-4e69-83ce-d58c39776812)
![image](https://github.com/GetPublii/Publii/assets/7956750/e64377f5-6d66-4df8-a1f7-13fd34ed1392)

**Comment by GetPublii** [comment]
It should work; the code is ok. Where do you edit the theme files? Provide the path to your theme.
Also, try restarting the app.
